src/main/fc/rc.c

```diff
@@ -148,6 +148,24 @@ float applyRaceFlightRates(const int axis, float rcCommandf, const float rcComma
     return angleRate;
 }
 
+float applyKissRates(const int axis, float rcCommandf, const float rcCommandfAbs)
+{
+    float kissRpyUseRates;
+    const float rcCurvef = currentControlRateProfile->rcExpo[axis] / 100.0f;
+
+    if (currentControlRateProfile->rates[axis] < 100) {
+        kissRpyUseRates = 1 - rcCommandfAbs * currentControlRateProfile->rates[axis] / 100.0f;
+    } else {
+        //prevent division by zero
+        kissRpyUseRates = 1 - rcCommandfAbs * 0.99f;
+    }
+    
+    rcCommandf = (power3(rcCommandf) * rcCurvef + rcCommandf * (1 - rcCurvef)) * (currentControlRateProfile->rcRates[axis] / 1000.0f);
```

Comment:
It's a bit confusing to reassign the `rcCommandf` variable. It's an input parameter that defines the stick deflection amount.

Reply:
Done